src/index.html

```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="get"
+      class="search">
```

Comment:
```suggestion
    <form
      action="#"
      method="get"
      class="search"
    >
```
closing bracket should be on the same line as opening one